libraries/aws_vpcs.rb

```diff
@@ -43,7 +45,13 @@ def fetch_data
                    state: vpc[:state],
                    is_default: vpc[:is_default],
                    instance_tenancy: vpc[:instance_tenancy],
-                   tags: map_tags(vpc[:tags]) }]
+                   tags: map_tags(vpc[:tags]),
+                   cidr_block_association_set: {
+                     cidr_block_association_set_association_ids: vpc[:cidr_block_association_set].first[:association_id],
+                     cidr_block_association_set_statuses: vpc[:cidr_block_association_set].first[:cidr_block_state][:state],
+                     cidr_block_association_set_status_messages: vpc[:cidr_block_association_set].first[:cidr_block_state][:status_message],
+                   },
+                   ipv_6_cidr_block_association_set: {} }]
```

Comment:
should this be empty?

Reply:
All the functionalities of the ipv_6_cidr_block_association_set are already available in cidr_block_association_set. So if the customer wants it then we can give it.

Reply:
then why duplicate the key please remove it ! also remove it from [here](https://github.com/inspec/inspec-aws/pull/336/files?file-filters%5B%5D=.rb#diff-808fa2d7d990d30a91c8a9c550c9e2c275c9e908e59bee2a161362480c0be6acR25)

Reply:
as discussed we kept both the attributes as both attributes differs.